src/odb/src/db/dbSite.h

```diff
@@ -64,6 +64,7 @@ class _dbSite : public _dbObject
   uint _height;
   uint _width;
   dbId<_dbSite> _next_entry;
+  std::vector<std::pair<std::string, dbOrientType::Value>> _row_patterns;
```

Comment:
Please see dbDatabase.h for how schema updates are handled for backwards compatability.